source/_posts/2015-12-12-philips-hue-blocks-3rd-party-bulbs.markdown

```diff
@@ -19,7 +19,7 @@ Philips Hue FAQ entries regarding reversing the decision.
 <!--more-->
 Philips Hue was one of the first to get smart lights accepted by the mainstream. Their Zigbee-based hub is rock solid, never crashes, great API and worked with other Zigbee light bulbs too. They are a bit expensive but the platform was worth every penny, till now.
 
-Yesterday a thread on [/r/homeautomation][reddit-hue] published that Philips Hue now blocks all but their own bulbs and those of "friends of Hue". I have been able to confirm this in the [Philips Hue FAQ][philips-hue-faq] (Update Dec 14: they have removed the entries - [mirror here][philips-hue-faq-mirror]):
+Yesterday a thread on [/r/homeautomation][reddit-hue] published that Philips Hue now blocks all but their own bulbs and those of "friends of Hue". I have been able to confirm this in the ~~Philips Hue FAQ~~ (Update Dec 14: they have removed the entries - [mirror here][philips-hue-faq-mirror]):
```

Comment:
I don't think we need to strike it. Just removing the link from the text should be enough here.

```suggestion
Yesterday a thread on [/r/homeautomation][reddit-hue] published that Philips Hue now blocks all but their own bulbs and those of "friends of Hue". I have been able to confirm this in the Philips Hue FAQ (Update Dec 14: they have removed the entries - [mirror here][philips-hue-faq-mirror]):
```